modules/Components/src/client/PickList.coffee

```diff
@@ -361,7 +361,28 @@ class PickListSelect2Controller extends PickListSelectController
 			openOnEnter: false
 			allowClear: true
 			width: @width
-
+			ajax:
```

Comment:
This is most of the magic.  The select2 widget lets you add ajax calls for type ahead search.  I'm leveraging that.  I had to do this sort of hacky thing create a real URL to remove the "shortName" parameter if it existed in the list and only include the labelTextSearch and maxHits parameters.

Reply:
Indeed this is kinda hacky but it's reasonable. You're programmatically getting the base path of the URL, so this looks reliable.

Reply:
As a side note, this might be a way of fixing our relative vs full path urls problem with ACAS sometime int he future:
https://stackoverflow.com/questions/16553026/prepend-absolute-url-path-in-backbone-sync-for-an-api-on-another-server